src/CoreWCF.WebHttp/tests/RoutingTests.cs

```diff
@@ -37,6 +37,20 @@ public async Task NoParam()
             }
         }
 
+        [Fact]
+        public async Task NoParamWittEmptyRoot()
```

Comment:
@g7ed6e, for something small like this, you can actually edit the file yourself which adds a new commit to the PR source branch and then merge. If there's multiple or other work, I'll let the author handle it as it can make subsequent changes annoying (they have to pull down your commit from their fork).